### DISCUSSION TITLE
systemd-oomd for Fedora CoreOS

### DISCUSSION THREAD

**Discussion by JaimeMagiera** [okd-project/okd#663] [General]
The systemd-oomd service uses cgroups-v2 and pressure stall information (PSI) to monitor and take action on processes before an OOM occurs in kernel space.
https://man7.org/linux/man-pages/man8/systemd-oomd.service.8.html

In Fedora 34 systemd-oomd is enabled by default. 
https://fedoraproject.org/wiki/Changes/EnableSystemdOomd

FOCS is considering enabling by default as well to maintain parity. 
https://github.com/coreos/fedora-coreos-tracker/issues/840

What impact, if any, would there be on OKD?

**Comment by vrutkovs** [comment]
>systemd-oomd does not operate on systems that aren't cgroups v2

For some time this won't affect us. 

Later on we might experience random unexplained pod failures - kubelet uses kernel's OOM to issue OOMKilled event, so if the process is killed by systemd-oom it would look like unexplained SIGKILL sent to the process.

Seems we should be disabling that by default
